packages/sanes-chrome-extension/src/routes/signup/test/fillSignupForm.ts

```diff
@@ -56,6 +74,34 @@ export const submitAccountForm = async (
   await TestUtils.act(submitForm as any); //eslint-disable-line @typescript-eslint/no-explicit-any
 };
 
+export const handlePassPhrase2E = async (page: Page): Promise<void> => {
+  const checkbox = await page.$('input[type="checkbox"]');
+  expect(checkbox).not.toBeNull();
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  await checkbox!.click();
+
+  //await sleep(2000);
+  const mnemonic = await page.evaluate(
+    async (): Promise<string | null> => {
```

Comment:
I suggest:
```
const mnemonic = await page.evaluate(
    (): string | null => {
      const element = document.querySelector('p');
      if (!element) {
        return null;
      }

      return element.textContent;
    }
  );
```

Reply:
Also, have you considered the option to use: 
```
const mnemonicElem = page.$('p');
const mnemonic = mnemonicElem ? mnemonicElem.textContent || null
```

or something like that?

Reply:
Yes this solution is better, I think we should use `page` instance helper methods as far as possible instead  of `page.evaluate`.
But in this case we should call `page.evaluate` to get value of the element's text. So it is better to leave it as is. https://stackoverflow.com/questions/49388467/getting-property-from-elementhandle

Reply:
ok, sounds good to me. Just to let you know I was able to remove the `async` in my first proposed solution and it worked. Am I missing something?